integration-tests/e2e_tests_ci_tool/cmd/create_test_config_cmd.go

```diff
@@ -0,0 +1,190 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pelletier/go-toml/v2"
+	"github.com/spf13/cobra"
+
+	ctf_config "github.com/smartcontractkit/chainlink-testing-framework/config"
+)
+
+var createTestConfigCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create a test config from the provided flags",
+	Run: func(cmd *cobra.Command, _ []string) {
+		var tc ctf_config.TestConfig
+
+		var image, version, postgresVersion *string
+		if cmd.Flags().Changed(ChainlinkImageFlag) {
+			image = &oc.ChainlinkImage
+		}
+		if cmd.Flags().Changed(ChainlinkVersionFlag) {
+			version = &oc.ChainlinkVersion
+		}
+		if cmd.Flags().Changed(ChainlinkPostgresVersionFlag) {
+			version = &oc.ChainlinkPostgresVersion
+		}
+		if image != nil && version == nil || image == nil && version != nil {
+			fmt.Fprintf(os.Stderr, "Error: both chainlink-image and chainlink-version must be set\n")
+			os.Exit(1)
+		}
```

Comment:
what if both are `nil`? also, how does cobra treat empty string? what if I do `--chainlink-image=''`?

Reply:
@Tofel please see `cmd.Flags().Changed(ChainlinkVersionFlag)` . Only then the image value is used.